### ISSUE TITLE
 Bayesian Uncertainty Estimation for Batch Normalized Deep Networks

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#1328]
## 一言でいうと

予測の不確実性を推定する手法の提案。既存研究ではDropoutを使った不確実推定があったが、Dropoutよりもよく使われているBatch Normalizationを使い推定する。正規化に用いる平均/分散を学習データからサンプリングすることでモデル出力の分布を得る。

![image](https://user-images.githubusercontent.com/544269/62024634-bb1d6f00-b210-11e9-915e-419e6f81bc46.png)

### 論文リンク

https://arxiv.org/abs/1802.06455

### 著者/所属機関

Mattias Teye, Hossein Azizpour, Kevin Smith

* School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology

### 投稿日付(yyyy/MM/dd)

2018/2/18

**Comment by icoxfog417** [comment]
解説スライド
https://www.slideshare.net/harmonylab/bayesian-uncertainty-estimation-for-batch-normalized-deep-networks
